config/settings.yml.example

```diff
@@ -3,7 +3,12 @@
 :database:
 
 # URL of the smart proxy, used for reporting back
-:callback_url: 'https://localhost:8443'
+:callback_url: 'http://localhost:9292'
+
+# SSL settings for client authentication against smart proxy.
+# :ssl_ca_file: ssl/foreman_ca.pem
+# :ssl_private_key: ssl/foreman_key.pem
+# :ssl_certificate: ssl/foreman_cert.pem
```

Comment:
These should beforeman_ssl_private_key, foreman_ssl_certificate and foreman_ssl_ca_file, right?